Build error trying to build falco 0.40.0 with MINIMAL_BUILD and USE_JEMALLOC

Build error trying to build falco on Alpine 3.21 since release 0.40.0 with settings:

 -DMINIMAL_BUILD=On
 -DUSE_JEMALLOC=On 

```
/home/alpine/kubezero/falco/src/falco-0.40.0/userspace/falco/stats_writer.cpp: In member function 'void stats_writer::collector::get_metrics_output_fields_additional(nlohmann::json_abi_v3_11_3::json&, double)':
/home/alpine/kubezero/falco/src/falco-0.40.0/userspace/falco/stats_writer.cpp:465:54: error: 'using std::__shared_ptr_access<stats_writer, __gnu_cxx::_S_atomic, false, false>::element_type = class stats_writer' {aka 'class stats_writer'} has no member named 'm_output_rule_metrics_converter'
  465 |                                            m_writer->m_output_rule_metrics_converter) {
      |                                                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/alpine/kubezero/falco/src/falco-0.40.0/userspace/falco/stats_writer.cpp:466:59: error: 'using std::__shared_ptr_access<stats_writer, __gnu_cxx::_S_atomic, false, false>::element_type = class stats_writer' {aka 'class stats_writer'} has no member named 'm_output_rule_metrics_converter'
  466 |                                                 m_writer->m_output_rule_metrics_converter
      |                                                           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[3]: *** [userspace/falco/CMakeFiles/falco_application.dir/build.make:653: userspace/falco/CMakeFiles/falco_application.dir/stats_writer.cpp.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/Makefile2:2928: userspace/falco/CMakeFiles/falco_application.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:2990: userspace/falco/CMakeFiles/falco.dir/rule] Error 2
make: *** [Makefile:1061: falco] Error 2
```

most likely due to `m_output_rule_metrics_converter` being exluded during MINIMAL builds ?
```
#if defined(__linux__) and !defined(MINIMAL_BUILD) and !defined(__EMSCRIPTEN__)
	std::unique_ptr<libs::metrics::libs_metrics_collector> m_libs_metrics_collector;
	std::unique_ptr<libs::metrics::output_rule_metrics_converter> m_output_rule_metrics_converter;
#endif
```
 
See: https://github.com/falcosecurity/falco/blob/master/userspace/falco/stats_writer.h#L153

--- a/userspace/falco/stats_writer.cpp
+++ b/userspace/falco/stats_writer.cpp
@@ -461,6 +461,7 @@ void stats_writer::collector::get_metrics_output_fields_additional(
 					        METRIC_VALUE_UNIT_MEMORY_BYTES,
 					        METRIC_VALUE_METRIC_TYPE_MONOTONIC,
 					        val);
+#if defined(__linux__) and !defined(MINIMAL_BUILD) and !defined(__EMSCRIPTEN__)
 					if(m_writer->m_config->m_metrics_convert_memory_to_mb &&
 					   m_writer->m_output_rule_metrics_converter) {
 						m_writer->m_output_rule_metrics_converter
@@ -469,6 +470,9 @@ void stats_writer::collector::get_metrics_output_fields_additional(
 					} else {
 						output_fields[metric.name] = metric.value.u64;
 					}
+#else
+					output_fields[metric.name] = metric.value.u64;
+#endif
 				}
 			}
 		}

